cyclonedx_py/parser/poetry.py

```diff
@@ -33,6 +35,7 @@ class PoetryParser(BaseParser):
 
     def __init__(
             self, poetry_lock_contents: str, use_purl_bom_ref: bool = False,
+            omit_category: Set[str] = set(),  # noqa: B006
```

Comment:
:x: do NOT ignore/suppress the bug.
FIX IT! 

this is a serious issue. Please learn how to avoid the "bug" 
and fix the implementation.